src/providers/provider_dota.ts

```diff
@@ -27,8 +26,10 @@ export default class DotaProvider extends Provider {
       const newPatches = [];
 
       // Discard the old patches
+      // eslint-disable-next-line no-plusplus
       for (let i = 0; i < patchList.length && patchList[i] !== lastPatch; i++) {
         newPatches.push(patchList[i]);
+        this.logger.warn(patchList[i]);
```

Comment:
Why is that logged? I'm pretty sure there is nothing to warn about here, but if there is, the logging message should be more helpful.

Reply:
weird, I don't remember adding this... will remove